res/drawable/network_vbl_logo.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="utf-8"?>
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="202.8775dp"
+    android:height="53.224998dp"
+    android:viewportWidth="202.8775"
+    android:viewportHeight="53.224998">
+
+    <group
+            android:translateX="-87.132679"
+            android:translateY="-234.32115">
+        <group>
+            <group>
+                <group>
+                    <group
```

Comment:
This is quite some nested groups. Some XML creators allow you to flatten those. Could you please try?